src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
```

Comment:
Creating new objects like this cause is not a good practice. In the future, you should use the keyword 'new' every time. To avoid this situation, you can create a static final int variable, for example, NUMBER_OF_BALLS with a number of balls, in the Application class.
In the main method, create a list using the Ball class with the parameter NUMBER_OF_BALLS. Then, use the 'for' loop to create Lottery objects.